Build-Status-(release-1.10).md

```diff
@@ -1,19 +1,20 @@
 Currently showing the Build Status page for the `release/1.10` branch. You can also view the build status for other branches by clicking the links in the sidebar.
 
+
 # Rolling Builds
 
-For every commit we build and test of all of our build configurations on Windows 8.1 (Windows Server 2012 R2) using VS 2015.
+For every commit we build and test of all of our build configurations on Windows 10 with VS 2017 (Dev 15.7).
```

Comment:
nit: Is this true for arm?

Reply:
Not at the moment, but I'm not sure how much detail we should go into about issues like that. I'll try to update this. (I think I might have missed a daily config set to Windows 10 instead of Windows 8.1 in the `netci.groovy` script update. We'll see those issues come up soon enough if so, and we can address it. The daily build matrix is generally non-blocking so it's okay to update later.

Reply:
Added clarification.